### PR TITLE
new version 0.1.8

### PR DESCRIPTION
**Pull request by MustafaZain**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 
 gem 'multi_json'
-gem 'omniauth-oauth', '~> 1.0'
+gem 'omniauth', '~> 1.3', '>= 1.3.1'
```
Comment (roccoblues) [inline Gemfile]: Why is this needed?

Comment (MustafaZain) [inline Gemfile]: The `omniauth` gem is more updated and have continuous fix for issues recently than the `omniauth-oauth` last update was in 2015
